config/local.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Local environment settings
+ *
+ * Use this file to specify configuration settings for use while developing
+ * the app on your personal system.
+ *
+ * For more information, check out:
+ * https://sailsjs.com/docs/concepts/configuration/the-local-js-file
+ */
+
+module.exports = {
+  // Any configuration settings may be overridden below, whether it's built-in Sails
+  // options or custom configuration specifically for your app (e.g. Stripe, Sendgrid, etc.)
+
+  // config/sockets.js
+  onlyAllowOrigins: undefined,
+
+  // config/security.js
+  security: {
+    cors: {
+      allRoutes: true,
+      allowOrigins: ['http://localhost:1337', 'http://localhost:8080'],
+      allowCredentials: true,
+    },
+  },
```

Comment:
This seemed like we only needed it locally but wanted your feedback on if `development` env was sufficient.

Reply:
We should use development.js for this stuff. local settings should be removed from version control so they can be machine-specific ie if you want to set up a local postgres instance instead of mocking the db in memory you could configure it here without anyone else needing your same db setup

Reply:
We should use development.js for this stuff. local settings should be removed from version control so they can be machine-specific ie if you want to set up a local postgres instance instead of mocking the db in memory you could configure it here without anyone else needing your same db setup